jdeps broken for fast-jar layout due to brotli version in dependency chain

### Describe the bug

Running `jdeps` on the "getting-started" quickstart fails (for the default, fast-jar build layout) due to an issue with the specific version of brotli pulled in as a transitive dep. Specific `jdeps` invocation (with JDK17, JAVA_HOME defined appropriately):

```
project=path/to/quarkus-quickstarts/getting-started
jarfile="$project/target/quarkus-app/quarkus-run.jar"
libdir="$project/target/quarkus-app/lib"

# Create a temporary directory for a module path. Required so
# jdeps can resolve the java.xml.bind dependency
mkdir -p dependencies
shopt -s globstar
cp $libdir/**/*.jar dependencies

# Calculate dependencies
$JAVA_HOME/bin/jdeps \
    --ignore-missing-deps -R -s \
    --multi-release 17 \
    --module-path dependencies \
    "$jarfile" \
    "$libdir"/**/*.jar
```


### Expected behavior

This is the output when we use the `uber-jar` build layout, for reference. (This is considered a successful run)

```
getting-started-1.0.0-SNAPSHOT-runner.jar -> JDK removed internal API
getting-started-1.0.0-SNAPSHOT-runner.jar -> java.base
getting-started-1.0.0-SNAPSHOT-runner.jar -> java.compiler
getting-started-1.0.0-SNAPSHOT-runner.jar -> java.desktop
getting-started-1.0.0-SNAPSHOT-runner.jar -> java.logging
getting-started-1.0.0-SNAPSHOT-runner.jar -> java.management
getting-started-1.0.0-SNAPSHOT-runner.jar -> java.naming
getting-started-1.0.0-SNAPSHOT-runner.jar -> java.rmi
getting-started-1.0.0-SNAPSHOT-runner.jar -> java.transaction.xa
getting-started-1.0.0-SNAPSHOT-runner.jar -> java.xml
getting-started-1.0.0-SNAPSHOT-runner.jar -> jdk.compiler
getting-started-1.0.0-SNAPSHOT-runner.jar -> jdk.unsupported
getting-started-1.0.0-SNAPSHOT-runner.jar -> not found
```

### Actual behavior


output:
```
Exception in thread "main" java.lang.module.FindException: Unable to derive module descriptor for dependencies/com.aayushatharva.brotli4j.native-linux-x86_64-1.8.0.jar
	at java.base/jdk.internal.module.ModulePath.readJar(ModulePath.java:648)
	at java.base/jdk.internal.module.ModulePath.readModule(ModulePath.java:331)
	at java.base/jdk.internal.module.ModulePath.scanDirectory(ModulePath.java:284)
	at java.base/jdk.internal.module.ModulePath.scan(ModulePath.java:232)
	at java.base/jdk.internal.module.ModulePath.scanNextEntry(ModulePath.java:190)
	at java.base/jdk.internal.module.ModulePath.findAll(ModulePath.java:166)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsConfiguration$Builder.build(JdepsConfiguration.java:558)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsTask.buildConfig(JdepsTask.java:603)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsTask.run(JdepsTask.java:557)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsTask.run(JdepsTask.java:533)
	at jdk.jdeps/com.sun.tools.jdeps.Main.main(Main.java:49)
Caused by: java.lang.IllegalArgumentException: com.aayushatharva.brotli4j.native.linux.x86.64: Invalid module name: 'native' is not a Java identifier
	at java.base/jdk.internal.module.Checks.requireModuleName(Checks.java:52)
	at java.base/java.lang.module.ModuleDescriptor$Builder.<init>(ModuleDescriptor.java:1515)
	at java.base/java.lang.module.ModuleDescriptor.newAutomaticModule(ModuleDescriptor.java:2395)
	at java.base/jdk.internal.module.ModulePath.deriveModuleDescriptor(ModulePath.java:508)
	at java.base/jdk.internal.module.ModulePath.readJar(ModulePath.java:644)
	... 10 more

```

### How to Reproduce?

Reproducer:

1. build quarkus-quickstarts/getting-started from tag 2.16.4.Final
2. copy the script from the top of this bug report
3. adjust the `$project` variable to point at the quarkus-quickstarts clone
4. run script

### Output of `uname -a` or `ver`

Linux coil 6.2.0-rc2+ #5 SMP PREEMPT_DYNAMIC Mon Mar  6 15:11:54 GMT 2023 x86_64 GNU/Linux

### Output of `java -version`

openjdk version "17.0.6" 2023-01-17

### GraalVM version (if different from Java)

n/a

### Quarkus version or git rev

2.16.4.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.8.7

### Additional information

I chased this down a little bit. I think the issue is in Brotli, and was fixed here: https://github.com/hyperxpro/Brotli4j/pull/59 which went into 1.9.0.> Brotli is pulled in via (I think) Netty. Bumping the Brotli dependency to >= 1.9.0 might be enough to address this.

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -143,7 +143,7 @@
         <infinispan.protostream.version>4.6.1.Final</infinispan.protostream.version>
         <caffeine.version>3.1.1</caffeine.version>
         <netty.version>4.1.87.Final</netty.version>
-        <brotli4j.version>1.8.0</brotli4j.version>
+        <brotli4j.version>1.11.0</brotli4j.version>
         <reactive-streams.version>1.0.4</reactive-streams.version>
         <jboss-logging.version>3.5.0.Final</jboss-logging.version>
         <mutiny.version>2.1.0</mutiny.version>

